lib/osv/models.py

```diff
@@ -505,44 +505,42 @@ def _get_pre_0_8_affects(self):
       # Convert flattened events to range pairs (pre-0.8 schema).
       # TODO(ochang): Remove this once all consumers are migrated.
       # pylint: disable=cell-var-from-loop
-      new_range = lambda: vulnerability_pb2.AffectedRange(
+      new_range = lambda x, y: vulnerability_pb2.AffectedRange(
```

Comment:
nit: I think it's preferred to define functions rather than assign variables to lambdas.